### DISCUSSION TITLE
Settings for db location

### DISCUSSION THREAD

**Discussion by Mahantor** [PasteBar/PasteBarApp#165] [Q&A] [answered]
#### Q: The settings for this software are stored in which location on Windows?

**Comment by kurdin** [comment, marked as answer]
#### Q: The settings for this software are stored in which location on Windows?
@Mahantor  there is no settings yet, but you can look for a file name `pastebar-db.data` in your home folder and you can copy this file to another location or clone to create a backup. 
I'll see if I can add an option to move that file to user specific location using settings 

#### Q: Exporting, Importing, or Moving Boards and Configurations to Another Computer
https://github.com/orgs/PasteBar/discussions/225

**Comment by kurdin** [comment]
Custom db location user setting implemented in #283
